ext/redhat/pe-bolt-server.init

```diff
@@ -15,14 +15,14 @@
 prefix='/opt/puppetlabs/server/apps/bolt-server'
 export GEM_HOME="${prefix}/lib/ruby/gems/2.4.0"
 exec="${prefix}/bin/bolt-server"
-prog="pe-bolt-server"
+prog="bolt-server"
```

Comment:
`prog` was only used for lockfile/piddir/logdir, so I changed it to be consistent with packaging.